src/api/app/models/bs_request.rb

```diff
@@ -150,11 +150,16 @@ def self.actions_summary(payload)
     ret.join(', ')
   end
 
-  def self.new_from_xml(xml)
+  def self.new_from_xml(xml, is_create: false)
     hashed = Xmlhash.parse(xml)
 
     raise SaveError, 'Failed parsing the request xml' unless hashed
 
+    if is_create
+      raise SaveError, 'Request ID attribute not allowed when creating a request' if hashed['id']
```

Comment:
Also not sure if you need to raise an exception or just silently ignore it? I think it is reasonable when you create a new request to just ignore the id, no?